lib/util/cmcd_manager.js

```diff
@@ -410,6 +443,20 @@ shaka.util.CmcdManager = class {
       data.su = this.buffering_;
     }
 
+    if (data.v === 2 && this.playerInterface_.isLive()) {
+      data.ltc = this.playerInterface_.getLiveLatency();
+    }
+
+    if (
+      data.v === 2 &&
```

Comment:
What are these `2`?
Probably best to have them in a constant somewhere, so we can avoid magic numbers.
If it is the version, maybe something like `VERSION_2`.